### DISCUSSION TITLE
essay/2023/10/06/acquisition-is-hard

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#134] [Comments]
# essay/2023/10/06/acquisition-is-hard

전단지를 돌리는 게 우습다는 사람들이 있습니다. 네이버 카카오 나와서 겨우 전단지를 돌리고 있냐고.

https://jeho.page/essay/2023/10/06/acquisition-is-hard.html

**Comment by unnamedw** [comment]
저런 사람들 중 태반이 보통 간단한 전단지 하나도 안돌려본 사람들일 겁니다. 세상이 쉽지 않다는 걸 알수록 겸손하거든요.

**Comment by burndogfather** [comment]
인상깊은 글이네요.
제 블로그에 함께 읽으면 좋은 글로 링크합니다.
개발자가 왜 쪽팔리게 이런거까지 해야되? - https://burndogfather.com/272

**Comment by websocketbear** [comment]
언제부터 개발자라는 직업이 조선시대의 사대부와 같아 보여 이런 모습이 좀 안타까워요. 같은 서비스를 개발해도 힘든것도 같이 나누고, 쪽팔리는 것(?) 같이 나눌 수 있는 동료가 정말 좋은 거 같아요.

**Reply by BenjaminKim** [reply]
힘든 것, 쪽팔린 것 같이 나눠주는 동료는 어떤 직군에서든 만나기 힘든 특별한 동료입니다.
그런 동료를 만나게 된다면... 엄청난 행운으로 생각하고 꽉 잡으셔야 합니다. 😁
